app/javascript/controllers/pager_controller.js

```diff
@@ -0,0 +1,44 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  static targets = ["page", "prev", "next"]
+  static values = {
+    currentPosition: { type: Number, default: 0 },
+    range: { type: Number, default: 6 },
+    step: { type: Number, default: 1 },
+  }
+
+  currentPositionValueChanged() {
+    this.pageTargets.forEach((el) => {
+      if (this.inRange(parseInt(el.getAttribute("index")))) {
```

Comment:
so many parentheses lol. wasn't sure if `parseInt` was necessary here. sometimes i forget if strings/ints have to be strictly compared or not in JS.

Reply:
You could also ditch the index attribute, and just get the index from the forEach. Can be inferred from their order in the DOM anyways.

```
this.pageTargets.forEach((el, index) => {
```

Reply:
oh right, JS iterating magic. will update.